app/controllers/payment_status_controller.js

```diff
@@ -19,7 +19,7 @@ module.exports = (req, res) => {
   const product = req.product
   const payment = req.payment
   const data = {
-    currentServiceName: product.serviceName
+    serviceName: product.serviceName
```

Comment:
I don’t think this really need renaming but if you’ve updated it everywhere, it’s fine

Reply:
All the other pages were passingit as serviceName, it’s how the service name in the banner is populated. So I either renamed it or passed the same thing twice with two different names.